polaris-react/src/components/AppProvider/global.css

```diff
@@ -93,8 +93,6 @@ html {
   }
   scrollbar-width: thin;
   scrollbar-color: var(--p-color-bg) var(--p-color-bg);
-  transition: scrollbar-color var(--p-motion-duration-100)
```

Comment:
Hmm I thought this transition did work... maybe only in certain browser or with specific vendor prefixes?